src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,43 @@
 
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	private static final int STACK_SIZE = 12;
+    private static final int STACK_VALUE = -1;
 
-	public int total = -1;
+    private final int[] numbers = new int[STACK_SIZE];
 
-	public void countIn(int in) {
+    private int total = STACK_VALUE;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)